diagnosis/scripts/updateconfiguration.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash -e
+
+
+
+
+
+agentScript() {
+    cat <<EOF > agent.sh
+#!/bin/bash -e
+
+# cloud provider
+sudo cp /etc/kubernetes/azure.json /etc/kubernetes/azure.bak
+sudo jq '.cloudProviderRatelimit = true | .cloudProviderRateLimitQPS = 3 | .cloudProviderRateLimitBucket = 10 | .cloudProviderBackoffRetries = 1 | .cloudProviderBackoffDuration = 30' /etc/kubernetes/azure.bak | sudo dd status=none of=/etc/kubernetes/azure.json
+
+# kubelet
+sudo cp /etc/default/kubelet /etc/default/kubelet.bak
+sudo sed -i -e 's/--node-status-update-frequency=[0-9]*[a-z]/--node-status-update-frequency=1m/' /etc/default/kubelet
+
+# restart
+sudo systemctl daemon-reload
+sudo systemctl restart kubelet
+
+echo "=> azure.json updates"
+sudo grep -E 'cloudProviderRatelimit"|cloudProviderRateLimitQPS"|cloudProviderRateLimitBucket"' /etc/kubernetes/azure.json
+echo "=> kubelet restarted"
+systemctl status kubelet --no-pager -l
```

Comment:
on large clusters, a kubelet restart shows iptables exiting with status code 4
to make this script solid, we should retry if that happens

Reply:
For this I will open an issue. Which can go in later checkin.

Reply:
We are tracking this issue as https://github.com/msazurestackworkloads/azurestack-gallery/issues/198